### PR TITLE
[FF] Fix file format in the scapy project

### PR DESCRIPTION
**Pull request by gpotter2**
This PR breaks code history. Of course... but it will avoid any future `^M` in the existing files, or in new created ones :)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1,3 @@
 scapy/__init__.py	export-subst
+* text=auto
+*.bat text eol=crlf
```
Comment (gpotter2) [inline .gitattributes]: That makes .*bat files use CRLF, and all the others use LF

**Comment by guedou** [comment]
It seems good to me. @p-l- are you OK with this change ?

**Comment by p-l-** [comment]
@gpotter2 can you rebase against current master? Thanks!

**Comment by p-l-** [comment]
For some reason, Travis does not start and Appveryor fails...

**Comment by gpotter2** [comment]
@p-l- Yeah that's awful... https://github.com/secdev/scapy/issues/664

**Comment by p-l-** [comment]
OK. For Appveyor, in the meantime, I'm OK to merge PRs if you can run the tests on a Windows machine. For Travis I think we need to have the tests running (MacOS + Pypy + root / non root), so I guess we have to wait.

**Pull request merged**